### ISSUE TITLE
How can I add handsontable plugin in nuxt

### ISSUE THREAD

**Issue by sk31park** [handsontable/handsontable#9421]
I use handsontable in nuxt(vue2)
code here 👇

```
// handsontable.js
const HotTable = !process.client ? null : require('@handsontable/vue').HotTable
export default HotTable
```

```
// ...
import HotTable from '~/plugins/handsontable'

export default {
  name: 'Table',
  components: {
    HotTable,
  },
  props: ['data', 'index'],
  data() {
    return {
    }
  },
  mounted() {
  }
}
```

It's working but..
I want use plugin in handson table like 'manualRowMove', 'manualColumnMove'

How can I add this plugin..
There is no guide for nuxt in handsontable homepage.

**Comment by AMBudnik** [comment]
Hi @sk31park 

That is true, we do not have a dedicated Nuxt tutorial. But here https://stackoverflow.com/questions/60685462/how-to-implement-handsontable-into-nuxt-js I found the topic that can help. In this example, you pass all of the plugins/options that are going to be used within the `settings` object.

**Comment by sk31park** [comment]
I don't think the above example solves everything.
For example 'manualRowMove', 'manualColumnMove' option not work in above example setting.

I think It need to add each plug-in module.
I want to know how to add each module.
https://handsontable.com/docs/api/plugins/

**Comment by AMBudnik** [comment]
You are right, @sk31park 
I just created a sample project and got Handsontable working well. But I was not able to register any cell type or a plugin. I just kept getting the `window is not defined` error. I did not work with Nuxt before so even after reading some of the ideas on Stack Overflow I did not get any further than showing the base table.

I will ask my colleague to test it out. However, now he is engaged in one project that he can't postpone. I will update you as soon as I get their feedback.
